up42/order.py

```diff
@@ -140,7 +141,7 @@ def place(cls, auth: up42_auth.Auth, order_parameters: dict) -> "Order":
         Returns:
             Order: The placed order.
         """
-        url = host.endpoint(f"/v2/orders?workspaceId={auth.workspace_id}")
+        url = host.endpoint(f"/v2/orders?workspaceId={up42.main.workspace.workspace_id}")
```

Comment:
Is this expected, to call workspace_id now from the workspace singleton, right? Just wanted to confirm.

Reply:
No, we need to add workspace_id to function parameters